libvast/vast/system/actors.hpp

```diff
@@ -196,17 +196,22 @@ using query_supervisor_master_actor = typed_actor_fwd<
   caf::reacts_to<atom::worker, query_supervisor_actor>,
   caf::reacts_to<atom::worker, atom::wakeup, query_supervisor_actor>>::unwrap;
 
+/// The PARTITION CREATION LISTENER actor interface.
 using partition_creation_listener_actor = typed_actor_fwd<
-  caf::reacts_to<atom::update, partition_synopsis_pair>>::unwrap;
+  caf::reacts_to<atom::update, partition_synopsis_pair>,
+  caf::reacts_to<atom::update, std::vector<partition_synopsis_pair>>>::unwrap;
```

Comment:
Do we really need both handlers? It feels like we could easily remove the first one.

Reply:
I think for this PR we don't need both, but I anticipate that the signature of the second one has a good chance of changing again when we tackle sc-31848, so I opted to keep them separate to make it easier to disentangle them later.